public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -435,7 +435,7 @@ function Main (params) {
             if(!svl.taskContainer.hasMaxPriorityTask()) {
                 svl.neighborhoodModel.setNeighborhoodCompleteAcrossAllUsers();
             }
-
+            
```

Comment:
Can you remove this extra whitespace? It's always a good idea to check out what the diff for your PR looks like on Github so that you can go through and remove any accidental changes like this. When I think I've finished writing the code, I'll often click the "new pull request" button on Github and I won't actually click the "create pull request" button. Instead I'll just look at the diff, because Github PRs show the diff really nicely, and it helps me catch small errors like this.